### ISSUE TITLE
Will you consider supporting Hangar in addition to Spiget?

### ISSUE THREAD

**Issue by mbravorus** [itzg/docker-minecraft-server#2103]
### Enhancement Type

Not sure

### Describe the enhancement

Will you consider eventually supporting resource fetching from Hangar ( https://hangar.papermc.io/ ) in addition to Spiget?

**Comment by itzg** [comment]
That looks cool. IMHO that seems even better than Spiget since this seems to be under the PaperMC umbrella.

**Comment by itzg** [comment]
Making a note of API docs

https://hangar.papermc.io/api-docs

**Comment by L4R5** [comment]
+1

**Comment by Krymonota** [comment]
I just wanted to bring this to attention again. Since [Paper has announced its hard fork](https://forums.papermc.io/threads/the-future-of-paper-hard-fork.1451/) (i.e. is no longer dependent on Spigot), there will be a time when some Spigot plugins will no longer work on Paper and vice versa. This also means that more plugins for Paper will be released (exclusively/separately) on Hangar. The Purpur development team has already stated that they will go down the hard fork route together with Paper. No idea about Pufferfish, but most likely the same applies.

**Comment by itzg** [comment]
Thanks for the additional info @Krymonota 

It looks like Modrinth will continue as a viable plugin publishing platform even after the fork

![image](https://github.com/user-attachments/assets/e9100b34-1fd6-46a3-8a68-2cd993f1b028)

**Comment by Krymonota** [comment]
Yes, that's at least something 😄 But I'm afraid that some developers won't release their plugins on Modrinth, as it's more known in the community for mods than plugins and Paper also has its own platform with Hangar.

Maybe I or someone else could submit a pull request for Hangar support if you're open for that 🙂

**Comment by itzg** [comment]
PRs are always welcome. All I ask is that the Hangar support be implemented primarily in mc-image-helper, such as this

https://github.com/itzg/mc-image-helper/blob/master/src/main/java/me/itzg/helpers/modrinth/ModrinthCommand.java

**Comment by wackywobblewave** [comment]
> Yes, that's at least something 😄 But I'm afraid that some developers won't release their plugins on Modrinth, as it's more known in the community for mods than plugins and Paper also has its own platform with Hangar.
> 
> Maybe I or someone else could submit a pull request for Hangar support if you're open for that 🙂

+1 seen many plugins on hangar but not on modrinth
